server/database_schemas/schema.sql

```diff
@@ -391,7 +422,7 @@ Source: [https://utdallas.box.com/s/aa0wbsjdkpm7kuvrm5pxybhsg00svgi4](https://ut
   );
 
 insert into university_table(university_id, long_name, country_code, region, info_page_id, campus, housing, ranking) values
-  ('skku', 'Sungkyunkwan University', 'KOR', 'Seoul, Suwon', 'skku_page', "Suwon Campus" , 1, "145"),
+  ('skku', 'Sungkyunkwan University (SKKU)', 'KOR', 'Seoul, Suwon', 'skku_page', "Suwon Campus" , 1, "145"),
```

Comment:
Do we need '(SKKU)' for it to work properly?

Reply:
with the previous trigger, since every alphabet within names needed to match, (SKKU) was needed. However, with the changed trigger, I do not think so.